.travis.yml

```diff
@@ -10,11 +10,13 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u github.com/stretchr/testify
+  - go get -u github.com/GoASTScanner/gas
```

Comment:
To be consistent with our other repositories, we should use the HewlettPackard fork of this repository which doesn't introduce breaking changes as often as the GoASTScanner main repository: github.com/HewlettPackard/gas

See here for what I'm talking about:
https://github.com/Azure/azure-sdk-for-go/blob/master/rungas.sh#L8